eos/parameters/polarisations.yaml

```diff
@@ -5,6 +5,13 @@
     max:     +0.655
     comments: "cf. [PDG2012+]"
 
+# Decay parameter of the Lambda_c baryon from FOCUS collaboration
+"Lambda_c::alpha@FOCUS" :
```

Comment:
Comparing to the alpha parameter in hadronic Lamba decays, you appended ```@FOCUS```. I don't see why we need that? Convince me!